docs/application/native/guides/text-input/overview.md

```diff
@@ -25,6 +25,18 @@ You can use the following text input and voice features in your native applicati
 
   You can manage Voice-Control-Engine (VCE) service application and control the recorded audio. The result of audio recording is sent as a predefined command.
 
```

Comment:
Line 10: You can create an IME (Input Method Editor) application that provides a customized keyboard for the user. You can show and hide the keyboard as needed, and offer a keyboard option menu to allow the user to manage the keyboard settings.

According to the google style guide please change to:
Input Method Editor (IME)

Reply:
fixed.